python/rmm/rmm/python/__init__.pxd

```diff
@@ -11,10 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from libc.stdint cimport uintptr_t
-from libcpp cimport bool
-from libcpp.utility cimport pair
-from libcpp.vector cimport vector
-
-ctypedef pair[const char*, unsigned int] caller_pair
```

Comment:
I don't see `caller_pair` used anywhere, so I deleted it

Reply:
So is this empty file needed?

Reply:
I checked the [docs](https://cython.readthedocs.io/en/stable/src/tutorial/pxd_files.html#init-pxd) for `__init__.pxd` and it looks like its used for shared declarations used in the cython package (which makes sense since its the cython equivalent of `__init__.py`). I'll delete for now unless anyone anticipates shared declarations in the future. Even then, we could always add it back.